src/content/docs/pt-br/tutorial/5-astro-api/1.mdx

```diff
@@ -0,0 +1,247 @@
+---
+type: tutorial
+title: Crie um arquivo de postagens do blog
+description: |-
+  Tutorial: Construa seu primeiro blog Astro —
+  Utilize Astro.glob() para acessar dados de arquivos no seu projeto
+i18nReady: true
+---
+import Box from '~/components/tutorial/Box.astro';
+import Checklist from '~/components/Checklist.astro';
+import MultipleChoice from '~/components/tutorial/MultipleChoice.astro';
+import Option from '~/components/tutorial/Option.astro';
+import PreCheck from '~/components/tutorial/PreCheck.astro';
+
+
+Agora que você tem algumas postagens do blog com links para, vamos configurar a página Blog para criar uma lista de postagens automaticamente!
```

Comment:
```suggestion
Agora que você tem algumas postagens do blog para adicionar links para, vamos configurar a página Blog para criar uma lista de postagens automaticamente!
```

Reply:
i think "Agora que você tem algumas postagens do blog com links para" was a bit off when i was reading it, when comparing with the original: "blog posts to link to"

 but i  am also not sure if my suggestion is a better approach to translate the term, please feel free to think about this one haha

Reply:
I think your suggestion works great, in Portuguese the problem is really the fact we can't use "link" as a verb, so we have to use an auxiliary verb to make it work, so it can feel a bit off sometimes, thanks!